Hi @gante !

The PR #33219 has a change

```
        # Special case: some models have generation attributes set in the decoder. Use them if still unset in the
        # generation config (which in turn is defined from the outer attributes of model config).
        decoder_config = model_config.get_text_config(decoder=True)
```
which cause 

```bash
        config_dict = model_config.to_dict()
        # generation config (which in turn is defined from the outer attributes of model config).
>       decoder_config = model_config.get_text_config(decoder=True)
E       AttributeError: 'GenerationConfig' object has no attribute 'get_text_config'

src/transformers/generation/configuration_utils.py:1240: AttributeError
================================================================================ short test summary info =================================================================================FAILED tests/generation/test_configuration_utils.py::GenerationConfigTest::test_initialize_new_kwargs - AttributeError: 'GenerationConfig' object has no attribute 'get_text_config' 
```

Could you take a look 🙏 ?

--- a/.circleci/create_circleci_config.py
+++ b/.circleci/create_circleci_config.py
@@ -312,6 +312,15 @@ repo_utils_job = CircleCIJob(
 )
 
 
+non_model_job = CircleCIJob(
+    "non_model",
+    docker_image=[{"image": "huggingface/transformers-torch-light"}],
+    marker="not generate",
+    parallelism=6,
+    pytest_num_workers=8,
+)
+
+
 # We also include a `dummy.py` file in the files to be doc-tested to prevent edge case failure. Otherwise, the pytest
 # hangs forever during test collection while showing `collecting 0 items / 21 errors`. (To see this, we have to remove
 # the bash output redirection.)
@@ -336,7 +345,7 @@ doc_test_job = CircleCIJob(
     pytest_num_workers=1,
 )
 
-REGULAR_TESTS = [torch_and_tf_job, torch_and_flax_job, torch_job, tf_job, flax_job, hub_job, onnx_job, tokenization_job, processor_job, generate_job] # fmt: skip
+REGULAR_TESTS = [torch_and_tf_job, torch_and_flax_job, torch_job, tf_job, flax_job, hub_job, onnx_job, tokenization_job, processor_job, generate_job, non_model_job] # fmt: skip
 EXAMPLES_TESTS = [examples_torch_job, examples_tensorflow_job]
 PIPELINE_TESTS = [pipelines_torch_job, pipelines_tf_job]
 REPO_UTIL_TESTS = [repo_utils_job]

--- a/utils/tests_fetcher.py
+++ b/utils/tests_fetcher.py
@@ -1148,21 +1148,19 @@ JOB_TO_TEST_FILE = {
     "pipelines_torch": r"tests/models/.*/test_modeling_(?!(?:flax_|tf_)).*",
     "tests_hub": r"tests/.*",
     "tests_onnx": r"tests/models/.*/test_modeling_(?:tf_|(?!flax)).*",
-    "tests_non_model": r"tests/.*?/test_.*\.py",
+    "tests_non_model": r"tests/[^/]*?/test_.*\.py",
 }
 
 
 def create_test_list_from_filter(full_test_list, out_path):
     all_test_files = "\n".join(full_test_list)
     for job_name, _filter in JOB_TO_TEST_FILE.items():
-        print(job_name)
         file_name = os.path.join(out_path, f"{job_name}_test_list.txt")
         if job_name == "tests_hub":
             files_to_test = ["tests"]
         else:
             files_to_test = list(re.findall(_filter, all_test_files))
         print(job_name, file_name)
-        print(len(files_to_test))
         if len(files_to_test) > 0:  # No tests -> no file with test list
             with open(file_name, "w") as f:
                 f.write("\n".join(files_to_test))

--- a/utils/tests_fetcher.py
+++ b/utils/tests_fetcher.py
@@ -1148,6 +1148,7 @@ JOB_TO_TEST_FILE = {
     "pipelines_torch": r"tests/models/.*/test_modeling_(?!(?:flax_|tf_)).*",
     "tests_hub": r"tests/.*",
     "tests_onnx": r"tests/models/.*/test_modeling_(?:tf_|(?!flax)).*",
+    "tests_non_model": r"tests/.*?/test_\.py",
 }
 
 

--- a/utils/tests_fetcher.py
+++ b/utils/tests_fetcher.py
@@ -1148,7 +1148,7 @@ JOB_TO_TEST_FILE = {
     "pipelines_torch": r"tests/models/.*/test_modeling_(?!(?:flax_|tf_)).*",
     "tests_hub": r"tests/.*",
     "tests_onnx": r"tests/models/.*/test_modeling_(?:tf_|(?!flax)).*",
-    "tests_non_model": r"tests/.*?/test_\.py",
+    "tests_non_model": r"tests/.*?/test_.*\.py",
 }
 
 

